client/my-sites/stats/site.jsx

```diff
@@ -144,8 +147,29 @@ class StatsSite extends Component {
 		this.props.recordTracksEvent( 'calypso_stats_parsely_banner_click' );
 	};
 
+	buildPrivateSiteBanner() {
+		return (
+			<Banner
+				callToAction={ translate( 'Launch your site' ) }
+				className="stats__private-site-banner"
+				description={ translate(
+					'Changing your site from private to public helps people find you and get more visitors. Don’t worry, you can keep iterating on your site.'
```

Comment:
Nit: from a copy perspective, "iterating on" may not resonate with a lot of people. Maybe using "working on your site" or "improving your site" could work.

Reply:
Good point. Here's the alternative copy:
`Changing your site from private to public helps people find you and get more visitors. Don’t worry, you can keep working on your site.`